src/WebJobs.Script.Analyzers/Constants.cs

```diff
@@ -5,15 +5,22 @@ namespace Microsoft.Azure.Functions.Analyzers
 {
     internal static class Constants
     {
+        internal static class Assemblies
```

Comment:
nit: Can name the class as `AssemblyNames` and const as `WebJobs`. So it will be used as `AssemblyNames.WebJobs`

Reply:
This is less redundant; I like it.